src/mcmc/ess.jl

```diff
@@ -27,15 +27,20 @@ function DynamicPPL.initialstep(
     rng::AbstractRNG, model::Model, spl::Sampler{<:ESS}, vi::AbstractVarInfo; kwargs...
 )
     # Sanity check
-    vns = _getvns(vi, spl)
-    length(vns) == 1 ||
-        error("[ESS] does only support one variable ($(length(vns)) variables specified)")
-    for vn in only(vns)
+    # TODO(mhauru) What is the point of the first check? Why is it relevant that if there
```

Comment:
Before merging, double-check what is going on here.

Reply:
This has been removed now. I also added a test that checks that doing so is okay.